src/arche/rules/result.py

```diff
@@ -158,14 +185,13 @@ def get_detailed_messages_count(self):
             ]
         )
 
-    def show(self, short: bool = False, keys_limit: int = 10):
+    def show(self, keys_limit: int = 10):
         from arche.report import Report
 
+        report = Report()
         IPython.display.clear_output()
-        Report.write_summary(self)
-        Report.write_rule_details(self, short=short, keys_limit=keys_limit)
-        for f in self.figures:
-            f.show()
+        report.save(self)
+        report(self)
```

Comment:
```
IPython.display.clear_output()
Report(self)
```

Reply:
> ```
> IPython.display.clear_output()
> Report(self)
> ```

It's not possible to call Report(self) (this will call the constructor). I changed it to Report()(self)